include/oneapi/dpl/experimental/kt/internal/esimd_radix_sort_utils.h

```diff
@@ -813,7 +833,7 @@ struct __rng_pack
     static constexpr bool __has_values = !std::is_void_v<_ValT>;
 
     auto& __keys_rng() { return __m_keys_rng; }
-    auto __keys_acc() const { return __m_keys_rng.data(); }
+    auto __keys_acc() const { return __rng_data(__m_keys_rng); }
```

Comment:
Is this `__keys_acc()` function really required?
We may avoid it and call `__keys_acc(__keys_rng())`